### PR TITLE
Revisión de Templates

### PR DESCRIPTION
**Pull request by paocorrales**
@yabellini all 4 templates are included in this PR, enjoy!

### CODE REVIEW

**Comment by yabellini** [comment]
Thanks!!

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 1]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
 
 ```
 
 ```
 
-
 ## Alcance
 
-- Por favor, indica qué categoría(s) aplican a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories). Por favor, tilda todas las apropiadas. Si no estás seguro, te sugerimos que comiences un pre-envío.
+- Por favor, indica qué categoría(s) aplica(n) a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#categor%C3%ADas-de-paquetes). Por favor, tilda todas las apropiadas. Si tienes dudas, te sugerimos que comiences un pre-envío.
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:32]: Posiblemente esta url cambie.

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 2]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
-	- [ ] análisis de texto (text analysis)
+	- [ ] datos de texto (text data)
 
 - Explica cómo y por qué el paquete encaja dentro de estas categorías (1 a 3 oraciones):
 
-
 - ¿Cuál es la audiencia esperada y las aplicaciones científicas de este paquete?
 
-- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría (documento en Inglés)](https://ropensci.github.io/dev_guide/policies.html#overlap)?
+- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#overlap)?
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:52]: Mismo comentario que el anterio.

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 3]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
-
-- Si ya has hecho una consulta de pre-envío, por favor pega el enlace al _issue_ correspondiente, una publicación del foro, u otra discusión. Alternativamente, etiqueta al editor (con @tag) con el que te contactaste.
-
--   (Si aplica) Explique las razones de los [elementos `pkgcheck`](https://docs.ropensci.org/pkgcheck/) que su paquete no puede pasar.
+-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación con Sujetos Humanos_](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#%C3%A9tica-privacidad-de-los-datos-e-investigaci%C3%B3n-con-sujetos-humanos)?
 
+- Si ya has hecho una consulta de pre-envío, por favor pega el enlace al _issue_ correspondiente, una publicación del foro, u otra discusión. Alternativamente, etiqueta a la persona del equipo editorial (con @tag) con la que te contactaste.
 
+-   (Si aplica) Explique las razones por las cuales el paqute no satisface alguno de los [chequeos de `pkgcheck`](https://docs.ropensci.org/pkgcheck/).
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:58]:
```suggestion
-   (Si aplica) Explica las razones por las cuales el paquete no satisface alguno de los [chequeos de `pkgcheck`](https://docs.ropensci.org/pkgcheck/).
```

**Review by yabellini — APPROVED** [review]
Some questions.

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md (hunk 1 above)
Comment (yabellini) [reply .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:32]: Las cambiamos todas juntas si es que cambia.

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md (hunk 2 above)
Comment (yabellini) [reply .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:52]: Las cambiamos todas juntas si es que cambia.

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 4]
```diff
@@ -1,56 +1,49 @@
...
-	- [ ] herramientas para trabajo de campo y reproducibilidad (field and lab reproducibility tools)
-	- [ ] ligamientos con software de base de datos (database software bindings)
+	- [ ] capa de interfaz de software científico (scientific software wrappers)
+	- [ ] herramientas de reproducibilidad de campo y laboratorio (field and lab reproducibility tools)
+	- [ ] enlaces a software de bases de datos (database software bindings)
 	- [ ] datos geoespaciales (geospatial data)
-	- [ ] análisis de texto (text analysis)
-	
-     **Packages Estadisticos**
+	- [ ] datos de texto (text data)
```
[the remark this answers is not in the document]
Comment (yabellini) [reply .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:44]: @maelle alguna opinión sobre esto?

[the remark this answers is not in the document]
Comment (maelle) [reply .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:44]: I've opened https://github.com/ropensci/dev_guide/issues/648, excellent catch, muchas gracias!

**Comment by yabellini** [comment]
@mpadge @maelle can I merge this PR?

**Review by mpadge — CHANGES_REQUESTED** [review]
Thanks @yabellini for not losing sight of this important PR! Yes, you can merge it, but with one change of removing template G for stats, because we currently have no actual ability to handle Spanish stats submissions. Could you please remove from this PR, merge the rest, and then open another one with just that template from an internal branch of this repo? Then we'll have the template ready to go once we have developed that far. Thanks!

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md (hunk 1 above)
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:32]:
```suggestion
- Por favor, indica qué categoría(s) aplica(n) a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes](https://devguide.ropensci.org/es/softwarereview_policies.es.html#package-categories). Por favor, tilda todas las apropiadas. Si tienes dudas, te sugerimos que comiences un pre-envío.
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md (hunk 2 above)
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:52]:
```suggestion
- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría](https://devguide.ropensci.org/es/softwarereview_policies.es.html#overlap)?
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 5]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
+- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#overlap)?
 
--   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación de Sujetos Humanos_ (documento en Inglés)](https://devguide.ropensci.org/policies.html#ethics-data-privacy-and-human-subjects-research)?
-
-- Si ya has hecho una consulta de pre-envío, por favor pega el enlace al _issue_ correspondiente, una publicación del foro, u otra discusión. Alternativamente, etiqueta al editor (con @tag) con el que te contactaste.
-
--   (Si aplica) Explique las razones de los [elementos `pkgcheck`](https://docs.ropensci.org/pkgcheck/) que su paquete no puede pasar.
+-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación con Sujetos Humanos_](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#%C3%A9tica-privacidad-de-los-datos-e-investigaci%C3%B3n-con-sujetos-humanos)?
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:54]:
```suggestion
-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación con Sujetos Humanos_](https://devguide.ropensci.org/es/softwarereview_policies.es.html#ethics-data-privacy-and-human-subjects-research)?
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 6]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
+-   (Si aplica) Explica las razones por las cuales el paquete no satisface alguno de los [chequeos de `pkgcheck`](https://docs.ropensci.org/pkgcheck/).
 
 ## Revisiones Técnicas
 
 Tilda los siguientes items para confirmar que los has completado:
 
-- [ ] He leído la [guía sobre paquetes de rOpenSci](https://devguide.ropensci.org/building.html).
-- [ ] He leído la [guía para autores](https://devdevguide.netlify.app/authors-guide.html) y pienso mantener este paquete durante al menos 2 años o encontrar un reemplazo.
-
+- [ ] He leído la [guía de empaquetado de rOpenSci](https://devdevguide.netlify.app/es/pkg_building.es.html).
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:64]:
```suggestion
- [ ] He leído la [guía de empaquetado de rOpenSci](https://devguide.ropensci.org/es/pkg_building.es.html).
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 7]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
+- [ ] He leído la [guía de autoría](https://devdevguide.netlify.app/es/softwarereview_author.es.html) y pienso mantener este paquete durante al menos 2 años o encontrar un reemplazo.
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:65]:
```suggestion
- [ ] He leído la [guía de autoría](https://devguide.ropensci.org/es/softwarereview_author.es.html) y pienso mantener este paquete durante al menos 2 años o encontrar un reemplazo.
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 8]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
 
 Este paquete:
 
 - [ ] No viola los Términos de Servicio de ningún servicio con los que interactúa.
 - [ ] Tiene una licencia aceptada por CRAN y OSI.
-- [ ] Contiene un [archivo README con instrucciones para instalar la versión de desarrollo](https://ropensci.github.io/dev_guide/building.html#readme).
-- [ ] Incluye [documentación con ejemplos para todas las funciones, creada con roxygen2](https://ropensci.github.io/dev_guide/building.html#documentation).
+- [ ] Contiene un [archivo README con instrucciones para instalar la versión de desarrollo](https://devdevguide.netlify.app/es/pkg_building.es.html#readme).
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:71]:
```suggestion
- [ ] Contiene un [archivo README con instrucciones para instalar la versión de desarrollo](https://devguide.ropensci.org/es/pkg_building.es.html#readme).
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 9]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
+- [ ] Incluye [documentación con ejemplos para todas las funciones, creada con roxygen2](https://devdevguide.netlify.app/es/pkg_building.es.html#documentaci%C3%B3n).
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:72]:
```suggestion
- [ ] Incluye [documentación con ejemplos para todas las funciones, creada con roxygen2](https://devguide.ropensci.org/es/pkg_building.es.html#documentation).
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 10]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
 - [ ] Contiene una viñeta (_vignette_) con ejemplos de sus funciones esenciales y su uso.
-- [ ] Tiene una [suite de tests (documento en Inglés)](https://devguide.ropensci.org/building.html#testing).
-- [ ] Tiene [integración contínua (documento en Inglés)](https://devguide.ropensci.org/ci.html), incluyendo reporte de cobertura de tests.
+- [ ] Tiene una [suite de tests](https://devdevguide.netlify.app/es/pkg_building.es.html#testeo).
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:74]:
```suggestion
- [ ] Tiene una [suite de tests](https://devguide.ropensci.org/es/pkg_building.es.html#testing).
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 11]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
+- [ ] Tiene [integración contínua](https://devdevguide.netlify.app/es/pkg_ci.es.html), incluyendo reporte de cobertura de tests.
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:75]:
```suggestion
- [ ] Tiene [integración contínua](https://devguide.ropensci.org/es/pkg_ci.es.html), incluyendo reporte de cobertura de tests.
```

File: .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md [hunk 12]
```diff
@@ -4,102 +4,95 @@ about: Usa esta plantilla para enviar un software a revisar en Español (Experim
...
-- (*Aunque no es requerido, recomendamos tener un manuscrito completamente preparado y en Inglés, al momento de enviar.*)
+- (*Aunque no es requerido, recomendamos tener un manuscrito completamente preparado y en Inglés, al momento de enviar el paquete.*)
 - (*Por favor, no envíes tu paquete de forma separada a Methods in Ecology and Evolution*)
 
 </details>
 
 ## Código de Conducta
 
-- [ ] Estoy de acuerdo en cumplir el [Código de Conducta de rOpenSci (documento en Inglés)](https://ropensci.github.io/dev_guide/policies.html#code-of-conduct) durante el proceso de revisión, y en mantener mi paquete si éste es aceptado.
+- [ ] Estoy de acuerdo en cumplir el [Código de Conducta de rOpenSci](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#code-of-conduct) durante el proceso de revisión, y en mantener mi paquete si éste es aceptado.
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/D-enviar-software-para-revision.md:98]:
```suggestion
- [ ] Estoy de acuerdo en cumplir el [Código de Conducta de rOpenSci](https://devguide.ropensci.org//es/softwarereview_policies.es.html#code-of-conduct) durante el proceso de revisión, y en mantener mi paquete si éste es aceptado.
```

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 13]
```diff
@@ -1,56 +1,49 @@
...
 
-## Alcance 
+## Alcance
 
-- Por favor, indica qué categoría(s) aplican a este paquete. 
-Las puedes encontrar en nuestras [políticas de inclusión de paquetes (Inglés)](https://ropensci.github.io/dev_guide/policies.html#package-categories) 
-y [categorías de paquetes estadistico (Inglés)](https://stats-devguide.ropensci.org/overview.html#overview-categories). 
-Por favor, tilda todas las apropiadas. 
-Si no estás seguro, te sugerimos que comiences un pre-envío.
+- Por favor, indica qué categoría(s) aplica(n) a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#categor%C3%ADas-de-paquetes) y [categorías de paquetes estadistico (Inglés)](https://stats-devguide.ropensci.org/overview.html#overview-categories). Por favor, tilda todas las apropiadas.
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:28]:
```suggestion
- Por favor, indica qué categoría(s) aplica(n) a este paquete. Las puedes encontrar en nuestras [políticas de inclusión de paquetes](https://devguide.ropensci.org/es/softwarereview_policies.es.html#package-categories) y [categorías de paquetes estadistico (Inglés)](https://stats-devguide.ropensci.org/overview.html#overview-categories). Por favor, tilda todas las apropiadas.
```

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 14]
```diff
@@ -63,12 +56,13 @@ Si no estás seguro, te sugerimos que comiences un pre-envío.
 
 - Explica cómo y por qué el paquete encaja dentro de estas categorías (1 a 3 oraciones):
 
-- Si esta enviando un paquete estadistico, [incorporó documentación de los estandares en su codigo utilizando el paquete **srr**](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr)?
+- Si estás enviando un paquete estadístico, ¿[incorporaste documentación de los estándares en su código utilizando el paquete **srr**](https://stats-devguide.ropensci.org/pkgdev.html#pkgdev-srr)?
+
+- ¿Cuál es la audiencia esperada y las aplicaciones científicas de este paquete?
 
-- ¿Cuál es la audiencia esperada y las aplicaciones científicas de este paquete? 
+- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#overlap)?
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:63]:
```suggestion
- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría](https://devguide.ropensci.org/es/softwarereview_policies.es.html#overlap)?
```

File: .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md [hunk 15]
```diff
@@ -63,12 +56,13 @@ Si no estás seguro, te sugerimos que comiences un pre-envío.
...
 
-- ¿Hay otros paquetes de R que logren el mismo objetivo? Si los hay, ¿cómo se diferencian del tuyo, o alcanzan nuestro [criterio del mejor de su categoría (documento en Inglés)](https://ropensci.github.io/dev_guide/policies.html#overlap)?
+-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación con Sujetos Humanos_](https://devdevguide.netlify.app/es/softwarereview_policies.es.html#%C3%A9tica-privacidad-de-los-datos-e-investigaci%C3%B3n-con-sujetos-humanos)?
```
Comment (yabellini) [inline .github/ISSUE_TEMPLATE/E-proponer-software-para-revision.md:65]:
```suggestion
-   (Si aplica) ¿Tu paquete cumple con nuestras [guías de _Ética, Privacidad de Datos e Investigación con Sujetos Humanos_](https://devguide.ropensci.org/es/softwarereview_policies.es.html#ethics-data-privacy-and-human-subjects-research)?
```

**Review by yabellini — APPROVED** [review]
Updating the links to the Spanish version of the book.

**Pull request merged**